torchgen/api/python.py

```diff
@@ -951,6 +951,9 @@ def argument_type_str_pyi(t: Type) -> str:
             )
         elif str(t.elem) == "float":
             ret = "Sequence[_float]"
+        elif str(t.elem) == "SymInt" and t.size == 1:
```

Comment:
Why limit to size of 1? Also see https://github.com/pytorch/pytorch/issues/99907 (reassigned it to you)

Reply:
Maybe I'm misreading this, but when the size is not 1, it should just be a sequence as before, no? So will be handled by the `else` block.

Reply:
Would single int be broadcastable to all dimension? For consistency with old behavior, `_int` was added as an acceptable type if `t.size is not None`, but here you are restricting it to `t.size == 1`. Also, you technically need to add `torch.Size` to the list